client/components/data/query-billing-data/index.jsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingBillingData } from 'state/billing-data/selectors';
+import { requestBillingData } from 'state/billing-data/actions';
+
+class QueryBillingData extends Component {
+	static propTypes = {
+		requestingBillingData: PropTypes.bool,
+		requestBillingData: PropTypes.func
+	};
+
+	componentWillMount() {
```

Comment:
Yeah, Fiber is not released yet but we should probably start updating these lifecycle methods to ease our future potential update. There's this gist that could clarify "what can we do where" https://gist.github.com/bvaughn/923dffb2cd9504ee440791fade8db5f9